app/src/main/java/com/nervousfish/nervousfish/activities/SettingsActivity.java

```diff
@@ -130,15 +138,16 @@ private void updateListPreference(final Preference preference, final String stri
      * immediately updated upon calling this method. The exact display format is
      * dependent on the type of preference.
      *
-     * @see #sBindPreferenceSummaryToValueListener
+     * @see #BIND_PREFERENCE_SUMMARY_TO_VALUE_LISTENER
```

Comment:
But @see works excellent! It's designed for this exact purpose. Why would I want to use something else?